lib/eslint/eslint.js

```diff
@@ -95,6 +95,9 @@ const { Retrier } = require("@humanwhocodes/retry");
 //------------------------------------------------------------------------------
 
 const FLAT_CONFIG_FILENAMES = [
+    "eslint.config.ts",
+    "eslint.config.mts",
+    "eslint.config.cts",
     "eslint.config.js",
     "eslint.config.mjs",
     "eslint.config.cjs"
```

Comment:
Any particular reason to prefer TypeScript configs over JavaScript configs? If both are present, then probably the TypeScript config was transpiled to JavaScript in the same location, and in that case we could import the JavaScript config without Jiti. But maybe I'm missing something, that's why I'm genuinely asking.

Reply:
Fair point, I will flip the order.